weaviate/gql/filter.py

```diff
@@ -53,6 +54,15 @@
 ]
 
 
+class MediaType(Enum):
+    IMAGE = "image"
+    AUDIO = "audio"
+    VIDEO = "video"
+    THERMAL = "thermal"
+    DEPTH = "depth"
+    IMU = "imu"
```

Comment:
I think this is something internal, so you could just use auto() instead of "image" etc

Reply:
ah, I forgot that they are used to set the media type in the query